service/src/main/java/uk/gov/hmcts/reform/fpl/service/PlacementService.java

```diff
@@ -126,6 +126,8 @@ public PlacementEventData savePlacement(CaseData caseData) {
 
         placement.setApplication(sealingService.sealDocument(applicationDocument));
 
+        placement.setPlacementUploadDateTime(time.now());
```

Comment:
missing test that covers this

Reply:
added in service/src/test/java/uk/gov/hmcts/reform/fpl/service/PlacementServiceTest.java